apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-edit-details/community-edit-details.component.ts

```diff
@@ -1,18 +1,20 @@
 import { DOCUMENT } from '@angular/common';
-import { Component, ElementRef, EventEmitter, Inject, OnInit, Output, ViewChild } from '@angular/core';
+import { Component, ElementRef, EventEmitter, Inject, OnDestroy, OnInit, Output, ViewChild } from '@angular/core';
 import { FormBuilder, Validators } from '@angular/forms';
 import { ActivatedRoute } from '@angular/router';
 import { CommunitiesService } from 'apps/commudle-admin/src/app/services/communities.service';
-import { ICommunity } from 'apps/shared-models/community.model';
 import { LibToastLogService } from 'apps/shared-services/lib-toastlog.service';
 import { GooglePlacesAutocompleteService } from 'apps/commudle-admin/src/app/services/google-places-autocomplete.service';
+import { ICommunity } from 'apps/shared-models/community.model';
+import { Subscription } from 'rxjs';
+import { SeoService } from 'apps/shared-services/seo.service';
```

Comment:
let's use @commmudle/shared-models for ICommunity and SeoService from @commudle/shared-service